src/ConfigProvider.php

```diff
@@ -15,11 +15,29 @@ final class ConfigProvider
     public function __invoke(): array
     {
         return [
-            'dependencies'       => $this->getDependencies(),
+            'dependencies' => $this->getDependencies(),
+            /**
+             * The top-level configuration key for defining custom view helpers.
+             *
+             * This option should use the `ServiceManagerConfiguration` array format
+             */
+            'view_helpers'       => [],
             'view_manager'       => [
+                /**
+                 * Encoding passed to the Escaper and possibly used in other view-related configuration
+                 */
```

Comment:
laminas-mvc does not set the encoding via the view manager. Can we move this to `view_helper_config`?